port_ocean/core/handlers/entity_processor/jq_entity_processor.py

```diff
@@ -70,22 +70,31 @@ async def _search_as_object(
     async def _calculate_entities(
         self, mapping: ResourceConfig, raw_data: list[dict[str, Any]]
     ) -> list[Entity]:
-        async def calculate_raw(data: dict[str, Any]) -> dict[str, Any]:
+        async def calculate_raw(data: dict[str, Any]) -> list[dict[str, Any]]:
             should_run = await self._search_as_bool(data, mapping.selector.query)
             if should_run and mapping.port.entity:
-                return await self._search_as_object(
-                    data, mapping.port.entity.mappings.dict(exclude_unset=True)
+                entity_mappings: dict[str, Any] = mapping.port.entity.mappings.dict(
+                    exclude_unset=True
                 )
-            return {}
+                if entity_mappings.get("items_to_parse"):
+                    items = await self._search(data, entity_mappings["items_to_parse"])
```

Comment:
there is data missing from the calculation pls take a look at how the webhook service does it